hardware/mifare/mifare_classic.ksy

```diff
@@ -0,0 +1,243 @@
+meta:
+  id: mifare_classic
+  endian: le
+  file-extension: mfd
+  title: "Mifare Classic RFID tag dump"
+  license: BSD-2-Clause
+doc-ref: |
+  https://github.com/nfc-tools/libnfc
+  https://www.nxp.com/docs/en/data-sheet/MF1S70YYX_V1.pdf
+seq:
+  - id: sectors
+    type: sector(_index)
+    repeat: eos
+types:
+  key:
+    seq:
+      - id: key
+        size: 6
+  sector:
+    params:
+      - id: idx
+        type: u1
+    seq:
+      - id: manufacturer
+        type: manufacturer
+        if: has_manufacturer
+      
+      - id: data_
+        -orig-id: abtData
+        size: _io.size - _io.pos - 16 # sizeof(trailer)
```

Comment:
You don't specify `size` for a `sector`, so this would essentially mean that there's always one sector, consisting of:

* `manufacturer`
* `data_` (spanning all up to last 16 bytes)
* `trailer` (16 bytes)

Are you sure that this is what you want?

Reply:
The problem right now is that your ksy always describes exactly 1 sector. Are you ok with that? If yes, why would you have an array of sectors?

Reply:
Ouch. This is definitely not OK, I need to think more about it, in fact I only used sectors, because each sector is read separately, maybe we really don't need that array. Thank you.

Reply:
fixed